### ISSUE TITLE
C# example would be nice

### ISSUE THREAD

**Issue by michael-hawker** [microsoftgraph/microsoft-graph-docs#4358]
Got to this page first from general web search on Calendar graph elements.

It'd be nice if this linked to or also had examples that utilized the [C# SDK](https://github.com/microsoftgraph/msgraph-sdk-dotnet).  Like how to utilize the `startDateTime` and `endDateTime` fields.
How do I use the C# SDK to perform these date filters?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8c927daf-9a45-a43a-2fae-5cec36564ea9
* Version Independent ID: 3af032d0-3e89-1b6c-be38-0da1e580d2e9
* Content: [List calendarView - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/calendar-list-calendarview?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/api/calendar-list-calendarview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/calendar-list-calendarview.md)
* Product: **outlook**
* Technology: **microsoft-graph**
* GitHub Login: @angelgolfer-ms
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Chikerenda** [comment]
Here is the example:
`			var queryOptions = new List<QueryOption>()
			{
				new QueryOption("startDateTime", fromDateTime.ToString(MicrosoftDateTimeFormat)),
				new QueryOption("endDateTime", toDateTime.ToString(MicrosoftDateTimeFormat))
			};
			IUserCalendarViewCollectionPage calendarView = await graphServiceClient.Me.CalendarView.Request(queryOptions).GetAsync();`

I'm using the constant for DateTime format:
`private const string MicrosoftDateTimeFormat = "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffffff";`

The DateTime format "o" will work only when DateTimeKind of your date is DateTimeKind.Unspecified.
So basically this code will also work:
`DateTime.SpecifyKind(toDateTime, DateTimeKind.Unspecified).ToString("o"))`

**Comment by lramosvea** [comment]
Closing this issue as C# code samples have been added to the doc already
